README.md

```diff
@@ -4,7 +4,12 @@ This is a seed project to get us started with prototyping.
 
 ## Installation
 
-First install the toolchain with [NPM](https://www.npmjs.org/):
+First install the command line tools globally with [NPM](https://www.npmjs.org/):
+```shell
+npm install --global gulp uglify-js2
```

Comment:
The `--global` flag is not necessary although it's probably more convenient to have `gulp` installed globally and not have to run it via npm.

The `uglify-js` dependency is already specified as a `devDependency` in the [package.json](https://github.com/forms-js/forms-js/blob/master/package.json) though, so it does not need to be added this way.

I'm not sure our instructions should recommend a global install...